packages/vuetify/src/components/VField/VField.sass

```diff
@@ -241,6 +241,7 @@
 /* region LABEL */
 .v-label.v-field-label
   contain: layout paint
+  display: block
```

Comment:
ellipsis doesn't work in default label `display:inline-flex`

Reply:
I'm guessing it was like that for a reason
#14359: 35b174de864c2fa3d5b9635e45314059bc68a3dc